observability/network_observability/configuring-operator.adoc

```diff
@@ -6,28 +6,29 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update the Flow Collector API resource to configure the Network Observability Operator and its managed components. The  Flow Collector is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`.
+You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference].
```

Comment:
Out of scope to your work, so it's up to you if you want to edit it, but per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#h) entry on "have to", it should only be used when "must" is too restrictive, which I don't believe is the case here:
```suggestion
You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it must be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference].
```